src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2FilteringEngine.java

```diff
@@ -97,12 +86,25 @@ public void applyStrandBiasFilter(final VariantContext vc, final Collection<Stri
         }
     }
 
-    public void applyEventDistanceFilters(final VariantContext vc, final Collection<String> filters) {
+    public static void applyEventDistanceFilters(final VariantContext vc, final Collection<String> filters) {
         final Integer eventCount = vc.getAttributeAsInt(GATKVCFConstants.EVENT_COUNT_IN_HAPLOTYPE_KEY, -1);
         if (eventCount >= 3) {
             filters.add(GATKVCFConstants.HOMOLOGOUS_MAPPING_EVENT_FILTER_NAME);
         }
     }
 
+    public static Set<String> calculateFilters(final M2ArgumentCollection MTAC, final VariantContext vc) {
```

Comment:
ideally it'd be cleaner if we can just concatenate the result of each filter in `filters`, instead of passing `filters` to each method and let it modify `filters` as a side effect. If you could put a TODO here I can give it a try